docs/admin-guide/load-config.md

```diff
@@ -1,125 +1,92 @@
-- **peddy_ped** path to a [peddy][peddy] ped file with an analysis of the pedigree based on variant information
-- **peddy_check** path to a [peddy][peddy] ped check file
-- **peddy_sex** path to a [peddy][peddy] ped sex check file
-- **multiqc** path to a [multiqc][multiqc] report with arbitrary information
-- **cnv_report** path to the CNV report file
-- **coverage_qc_report** path to static coverage and qc report file
-- **gene_fusion_report** path to a static file containing a gene fusion report produced by [Arriba][arriba]. Generated from default clinical data.
-- **gene_fusion_report_research** path to a static file containing a gene fusion report produced by [Arriba][arriba]. Generated from research data.
-- **default_gene_panels** list of default gene panels. Variants from the genes in the gene panels specified will be shown when opening the case in scout
-- **gene_panels** list of gene panels. This will specify what panels the case has been run with
-- **rank model version** which rank model that was used when scoring the variants
-- **SV rank model version** the SV rank model version used when scoring SV variants
-- **rank_score_treshold** only include variants with a rank score above this treshold
-- **analysis_date** time for analysis in datetime format. Defaults to time of uploading
-- **human_genome_build** what genome version was used.
-
-### Minimal config
+### Depreication Warnings:
+`bam_file`, `bam_path` and `alignment_path` are redundant in internal usage. Future versions of Scout will only
+support `alignment_path`.
+
+
+### Configuration Parameters
+Below are available configuration parameters for a Scout case. Names marked with asterix (*) are mandatory. 
+
+- **analysis_date(*)** Time for analysis in datetime format. Defaults to time of uploading. Example `2016-10-12 14:00:46`
+- **cnv_report** Path to the CNV report file.
+- **coverage_qc_report** Path to static coverage and qc report file.
+- **cohorts** List of strings, for meta organising study participants or cases.
+- **collaborators** list of collaborators
+- **coverage_qc_report** Path to html file with coverage and qc report.
+- **default_gene_panels** List of default gene panels. Variants from the genes in the gene panels specified will be shown when opening the case in scout
+- **delivery_report**: Path to html delivery report.
+- **family(*)**  Unique id of family in the configuration.
+- **gene_fusion_report** Path to a static file containing a gene fusion report produced by [Arriba][arriba]. Generated from default clinical data.
+- **gene_fusion_report_research** Path to a static file containing a gene fusion report produced by [Arriba][arriba]. Generated from research data.
+- **gene_panels** List of gene panels. This will specify what panels the case has been run with.
+- **human_genome_build** Version of genome version used.
+- **lims_id** Lims Id used.
+- **madeline** Path to a madeline pedigree file in xml format.
```

Comment:
mt-bam is missing in this list and I think also in the template. Could you add it?

It's the path to the reduced mitochondrial  bam/cram alignment file

Reply:
OK! Will do.